inst/rmarkdown/templates/xaringan/resources/uo.css

```diff
@@ -0,0 +1,103 @@
+.remark-slide-content {
+  border-top: 5px solid #007935;
+  border-bottom: 5px solid #007935;
+  font-size: 1.2em;
+  padding: 20px 60px;
+}
+
+.remark-slide-content > h1 {
+  margin-top: 20px;
+  margin-bottom: 0px;
+}
+
+.inverse {
+  background-color: #007935;
+  border:none;
+  font-color: #fff;
+  text-shadow: none;
+}
+
+.inverse, .inverse h1, .inverse h2, .inverse a {
+  color: #fff;
+}
+
+.inverse h3 {
+  color: #000;
+}
+
+.inverse code {
+  color: #E1E000;
+}
+
+.footnote {
+  border-top: 1px solid #007935;
+  font-size: 0.6em;
+}
+
+.remark-code-line-highlighted { background-color: rgba(225, 224, 0, 0.6); }
+
+h1, h2 {
+  color: #007935;
+}
+
+h3 {
+  color: #47484D;
+  margin: 0px;
+  padding: 0px;
+}
+
+code {
+  color: #007935;
+}
+
+a { 
+  color: #3F97B5;
+  text-decoration: none;
+  transition: all .2s ease-in-out;
+}
+
+a:hover {
+    font-size: 1.2em;
+    color: #EC008C;
+}
+
+.remark-slide-number {
+  font-size: 0.6em;
+}
+
+.title-slide {
+  background-image: url(https://github.com/uo-datasci-specialization/data_sci_specialization/raw/master/xaringan_uo_theme/UOSignature-107-4C.png);
```

Comment:
Here is my only concern: we need to make sure the repo owner (you) won't delete or move this image in the future.

Reply:
Thanks so much! Yes, that link could serve as a demo for the theme.

In terms of the image, I just committed a change with the link pointing to my personal repo where I can make sure it won't ever move. Does that work?

I also added myself as a contributor and updated the NEWS.md.